src/App.css

```diff
@@ -1,26 +1,44 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+a {
+    text-decoration: none;
+
+}
+
+
 .dark-theme {
     --color: white;
```

Comment:
We can handle the dark theme and light theme by setting different values for the color, background-color, and link color variables. We apply these values in the `.app` class